nemo/collections/nlp/metrics/classification_report.py

```diff
@@ -54,15 +55,28 @@ def validation_epoch_end(self, outputs):
         res: a torch.Tensor object with three elements: [true positives, false positives, false negatives].
     """
 
-    def __init__(self, num_classes: int, label_ids: Dict[str, int] = None):
-        super(ClassificationReport, self).__init__(name="ClassificationReport")
+    def __init__(
+        self,
+        num_classes: int,
+        label_ids: Dict[str, int] = None,
+        mode: str = 'macro',
+        dist_sync_on_step: bool = False,
+        process_group: Optional[Any] = None,
+    ):
```

Comment:
Could you please update docstring Args and example call based on the changes?

Reply:
done.